### PR TITLE
Snapshot page analyze dropdown menu for external tools

### PR DESCRIPTION
**Pull request by nellh**
Implements #3046

image

### CODE REVIEW

File: packages/openneuro-app/src/scripts/dataset/components/AnalyzeDropdown.tsx [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+export const AnalyzeDropdown: React.FC<CloneDropdownProps> = (
+  { datasetId, snapshotVersion },
+) => {
+  const brainlifeUrl =
+    `https://brainlife.io/openneuro/${datasetId}/${snapshotVersion}`
+  const nemarUrl =
+    `https://nemar.org/dataexplorer/detail?dataset_id=${datasetId}`
+  const cbrainUrl =
+    `https://portal.cbrain.mcgill.ca/openneuro/${datasetId}/versions/${snapshotVersion}`
```
Comment (effigies) [inline packages/openneuro-app/src/scripts/dataset/components/AnalyzeDropdown.tsx:26]: Looks like you found it. Sorry for completely forgetting about this.

File: packages/openneuro-app/src/scripts/dataset/components/AnalyzeDropdown.tsx [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+        <div className="dataset-git-access">
+          <span>
+            <h4>Analyze this dataset</h4>
+          </span>
+          <p>
+            Work with OpenNeuro datasets using any of these third-party tools.
+          </p>
+          <hr />
+          <img
+            src="https://brainlife.io/favicon.ico"
```
Comment (effigies) [inline packages/openneuro-app/src/scripts/dataset/components/AnalyzeDropdown.tsx]: Should we host our own copies, so they can rearrange their sites without breaking this?

**Review by effigies — APPROVED** [review]
Test failure will be a quick fix. One small question.

File: packages/openneuro-app/src/scripts/dataset/components/AnalyzeDropdown.tsx (hunk 2 above)
Comment (nellh) [reply packages/openneuro-app/src/scripts/dataset/components/AnalyzeDropdown.tsx]: There's trade offs either way but I think it does make sense. I'll make that change.

**Pull request merged**